yarn-project/noir-protocol-circuits/src/crates/types/src/abis/global_variables.nr

```diff
@@ -32,3 +57,14 @@ impl Hash for GlobalVariables {
             )
```

Comment:
Do we use this hash anywhere atm?

Reply:
We currently just used that in the old Header.block_hash function. There it won't be necessary once updated so I linked here the issue to nuke this.